network/message/authorization.go

```diff
@@ -1,15 +1,14 @@
 package message
 
 import (
-	"errors"
-	"fmt"
-
 	"github.com/onflow/flow-go/model/flow"
 	"github.com/onflow/flow-go/model/libp2p/message"
 	"github.com/onflow/flow-go/model/messages"
 	"github.com/onflow/flow-go/network/channels"
 )
 
+var AuthorizationConfigs map[string]MsgAuthConfig
```

Comment:
If possible, it would be better if this were package-private since it is mutable. I don't think we have a need to expose it publicly, given we have `GetMessageAuthConfig` for that.

Reply:
https://github.com/onflow/flow-go/pull/2648/commits/b5e0333740953c051441201837521280249a6341